### ISSUE TITLE
Priiloader needs the Framebuffer-Option

### ISSUE THREAD

**Issue by Senseless-Creature** [DacoTaco/priiloader#325]
Can you PLEASE add the Framebuffer-Option like USB-Loader GX has it in its Options? This would be so amazing- Wii Games are looking so awesome with the following Settings:

480P-Fix enabled
Deflicker forced off - extended
Framebuffer-Option enabled

Here you can see the difference
https://youtu.be/tzJth1XFCBU

Thank You so much for keeping Priiloader alive <3

**Comment by DacoTaco** [comment]
do you mean for loading games from discs? because for the priiloader menu itself im not sure what the use would be lol

**Comment by Senseless-Creature** [comment]
Yeah - maybe is it possible to set this system-wide for everything?

**Comment by Senseless-Creature** [comment]
Ok I see - can you add this Options for the SM and Disc-Drive?

**Comment by Weario** [comment]
I came across this thread while searching, so I'll just reply here.
I use this framebuffer width option too with USB Loader GX, it makes games slightly crisper as well.

Personally, I would like to see the framebuffer width option (as shown in the OP's video) in the Wii Menu too!
Mainly because I just like to see what the console is actually rendering without stretching (and thus blurrying) the images.

The deflicker option is there for the Wii system menu as well, might as well compliment it with the framebuffer width option too, if that's possible. =)
